pages/playlist/podcast/[id]/index.vue

```diff
@@ -48,14 +46,14 @@ const onPressShuffle = async () => {
             <ButtonStyled
               intent="primary"
               icon="icon.play"
-              @click.stop="onPressPlay"
+              @click="onPressPlay"
```

Comment:
(My understanding of .stop is that it also stops event propagation.)
In this case it doesn't make a difference since the parent is not clickable.
Personally I think @click is cleaner and we should only use @click.stop if there is a reason for it.

Reply:
I personally think that it's smarter to work the opposite way: To have @click.stop whenever the click-event consumes the click, which is the case most of the time. Otherwise we end up with weird side-effects because something closes because the user clicks something inside it. It seems harder to debug, than to have to open it up later.

Reply:
In the end, this is just what someone prefers, so I'll leave the code here as is.